lib/DDG/Spice/LaunchLibrary.pm

```diff
@@ -0,0 +1,31 @@
+package DDG::Spice::LaunchLibrary;
+# ABSTRACT: List upcoming rocket launches and where to watch them online
+
+use strict;
+use DDG::Spice;
+
+spice is_cached => 1;
+
+name "Launch Library";
+source "LaunchLibrary.net";
+description "List upcoming rocket launches and where to watch them online";
+primary_example_queries "upcoming rocket launches", "next space launch";
+category "reference";
+topics "geek", "science";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/LaunchLibrary.pm";
+attribution github => ["globalspin", "Chris Radcliff"],
+            twitter => "chris_radcliff";
+
+# Version 1.1 of the API allows GET requests. 
+# Parameters are included in the URL path; "/next/5" shows the next 5 launches
+# See launchlibrary.net for the API reference.
+spice to => 'http://launchlibrary.net/1.1/launch/next/5';
```

Comment:
I think we should be OK to bump this up little higher? 10 or 20 would be good if it's available.